cnxepub/tests/test_formatters.py

```diff
@@ -10,25 +10,124 @@
 import os
 import sys
 import unittest
+from lxml import etree
```

Comment:
FYI, not a big deal, but the import grouping here was correct... For future reference, things are typically grouped by 1) standard library 2) third-party 3) local.